src/components/AppNavigation/Settings/SettingsImportSection.vue

```diff
@@ -152,11 +151,32 @@ export default {
 				const lastModified = file.lastModified
 				const name = file.name
 				const size = file.size
-				const type = file.type
+				let type = file.type
+
+				// Handle cases where we are running inside a browser on Windows
+				//
+				// https://developer.mozilla.org/en-US/docs/Web/API/File/type
+				// "Uncommon" file-extensions will result in an empty type
+				// and apparently Microsoft considers calendar files to be "uncommon"
+				if (type === '') {
+					// If it's an xml file, our best guess is xCal: https://tools.ietf.org/html/rfc6321
+					// If it's a json file, our best guess is jCal: https://tools.ietf.org/html/rfc7265
+					// In every other case, our best guess is just plain old iCalendar: https://tools.ietf.org/html/rfc5545
+					if (name.endsWith('.xml')) {
+						type = 'application/calendar+xml'
+					} else if (name.endsWith('.json')) {
+						type = 'application/calendar+json'
```

Comment:
`JSCalendar` will be an issue when it comes around.

Reply:
Hm, Indeed. But as long as `JSCalendar` is still a draft and as long as we only support jCal anyway, i would not worry about it for now.